### PR TITLE
Remove data processing datasources

### PR DESCRIPTION
**Pull request by stephen-soltesz**
This change removes data sources from Grafana for the old data-processing cluster. The clusters and DNS names have all been removed from all projects.

Part of:
* https://github.com/m-lab/etl/issues/1092 

This change is [Reviewable](https://reviewable.io/reviews/m-lab/prometheus-support/1009)

### CODE REVIEW

**Comment by stephen-soltesz** [comment]
I stopped the build to prevent stepping on other changes in sandbox. I won't merge until it's been deployed and verified in sandbox.

@nkinkade IIRC, there are manual steps to delete data sources (is that correct?)

**Review by nkinkade — APPROVED** [review]
The official way to delete a provisioned datasource is by adding the datasource to a special config array that Grafana reads:

https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources

You can add the datasources to this file:

https://github.com/m-lab/prometheus-support/blob/main/config/federation/grafana/provisioning/datasources/deleted.yml

The unofficial way is outlined in the README:

https://github.com/m-lab/prometheus-support#deleting-provisioned-datasources


:lgtm:, when you are ready to merge.

> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/m-lab/prometheus-support/1009#-:-NdHW6SVCGsg7U_aoZgM:b-f242rq)* status: :shipit: complete! 1 of 1 approvals obtained

**Comment by stephen-soltesz** [comment]
Obsolete by: https://github.com/m-lab/prometheus-support/pull/1010

**Pull request closed without merging**
